lib/capybara/apparition/driver.rb

```diff
@@ -289,13 +286,14 @@ def pause
       # wait for data on STDIN or signal SIGCONT received
       keyboard = IO.select([read], nil, nil, 1) until keyboard || signal
 
-      unless signal
+      until signal
```

Comment:
I'm very cautious of the changes here and will need to find time to manually test them on multiple platforms